### ISSUE TITLE
PostCSS 6

### ISSUE THREAD

**Issue by dan-gamble** [Nitive/postcss-extract#225]
Would it be possible to update this for PostCSS 6? I tried it myself but can't for the life of me work out why it's not working.

On PostCSS 6 [extracted](https://github.com/Nitive/postcss-extract/blob/master/src/index.js#L70) here is empty, it seems that `rule.remove()` seems to remove the rule from `https://github.com/Nitive/postcss-extract/blob/master/src/index.js#L42` this node after you append it.

**Comment by Nitive** [comment]
Yes, it's definitely need to be done but I'm very busy right now, maybe I can found some time in the end of next week

**Comment by dan-gamble** [comment]
That's awesome, looking forward to it :) Would love to see what i was doing wrong as well
